src/OpenSage.Game/Logic/Object/Animation.cs

```diff
@@ -178,18 +178,26 @@ private enum AnimationDirection : ushort
 
 public enum AnimationType
 {
-    DefaultHeal,
-    StructureHeal,
-    VehicleHeal,
+    DefaultHeal = 0,
+    StructureHeal = 1,
+    VehicleHeal = 2,
+
+    // Demoralized = 3,
+
+    BombTimed = 4,
+    BombRemote = 5,
+    Disabled = 6,
+    BattlePlanBombard = 7,
+    BattlePlanHoldTheLine = 8,
+    BattlePlanSeekAndDestroy = 9,
+    Emoticon = 10,
+    Enthusiastic = 11,
+    Subliminal = 12,
+    CarBomb = 13,
+
     MoneyPickUp,
     LevelGainedAnimation,
     GetHealedAnimation,
-    BombTimed,
-    BombRemote,
-    CarBomb,
-    Disabled,
     AmmoFull,
     AmmoEmpty,
```

Comment:
out of curiosity, why do some of these not have values?

Reply:
The ones with values are the ones that appear [here](https://github.com/electronicarts/CnC_Generals_Zero_Hour/blob/0a05454d8574207440a5fb15241b98ad0b435590/GeneralsMD/Code/GameEngine/Include/GameClient/Drawable.h#L71-L107). I don't know this area well enough to know why some of these, like `MoneyPickup`, are actually used in Zero Hour but don't appear in this enum.